prime-router/src/test/kotlin/azure/ActionHistoryTests.kt

```diff
@@ -252,6 +261,7 @@ class ActionHistoryTests {
 //            connection.transaction(block = any() as TransactionalRunnable)
         }
 //        confirmVerified(connection)
+        TODO("JUnit -> AssertK")
```

Comment:
?TODO??

Reply:
@jorg3lopez - what is the purpose of the todo here?

Reply:
I'll take out the TODO. This test case doesn't run as it has the @test annotation commented out.

Reply:
Follow up: made the changes.